cms/envs/production.py

```diff
@@ -551,3 +551,38 @@ def get_env_setting(setting):
 ########################## Derive Any Derived Settings  #######################
 
 derive_settings(__name__)
+
+DJANGO_DEBUG = ENV_TOKENS.get('DJANGO_DEBUG', False)
+
+if DJANGO_DEBUG:
+    INSTALLED_APPS += ['debug_toolbar']
+    INTERNAL_IPS = ('127.0.0.1',)
```

Comment:
Currently, we don't utilize `ENV_TOKENS` in common.py. We have set a new env variable `ACTIVE_DJANGO_DEBUG_TOOLBAR` for dev only so that toolbar middleware doesn't get added on stage and prod.